docs/configuration/traces-config.md

```diff
@@ -145,6 +145,19 @@ scrape_configs:
 # `update` only modifies an existing k/v and `insert` only appends if the k/v
 # is not present. `upsert` does both.
 [ prom_sd_operation_type: <string> | default = "upsert" ]
+# Configures what methods to use to do association between spans and pods.
+# PromSD processor matches the IP address of the metadata labels from the k8s API
+# with the IP address obtained from the specified pod association method.
+# If a match is found then the span is labeled.
+#
+# Options are `ip`, `net.host.ip`, `k8s.pod.ip`, `hostname` and `connection`.
+#   - `ip`, `net.host.ip` and `k8s.pod.ip`, `hostname` match spans tags.
+#   - `connection` inspects the context from the incoming requests (gRPC and HTTP).
+#
+# By default, all methods are enabled, and evaluated in the order specified above.
```

Comment:
Shouldn't `k8s.pod.ip` have precedence over `ip`?

Reply:
Maybe? I don't know.`k8s.pod.ip` is a new tag for us, so I'm not familiar with it.

The current order is based on the previous implementation. I thought it'd be better to make it a non-op if relabelling was already working for someone with `ip` or `net.host.ip`.

I'm happy to change the ordering to whatever is more common and useful to users though (if that's `k8s.pod.ip`).